mypy/stubgen.py

```diff
@@ -172,6 +172,11 @@ def find_module_path_and_all(module: str, pyversion: Tuple[int, int],
             raise SystemExit(
                 "Can't find module '{}' (consider using --search-path)".format(module))
         module_all = None
+    if not (
+        module_all is None or
+        (isinstance(module_all, list) and all(isinstance(v, str) for v in module_all))
+    ):
```

Comment:
This statement is a bit hard to read, I think it would be easier if you reformulate it as `if module_all is not None and <bad __all__...>`.